src/Stratis.Bitcoin.Features.Wallet/WalletTransactionHandler.cs

```diff
@@ -529,10 +529,15 @@ public TransactionBuildContext(Network network)
         public bool Shuffle { get; set; }
 
         /// <summary>
-        /// Optional data to be added as an extra OP_RETURN transaction output with Money.Zero value.
+        /// Optional data to be added as an extra OP_RETURN transaction output.
         /// </summary>
         public string OpReturnData { get; set; }
 
+        /// <summary>
+        /// Optional amount to add to the OP_RETURN transaction output to pass the stratisX standardness (dust) check.
```

Comment:
```suggestion
         /// Optional amount to add to the OP_RETURN transaction output.
```

What if someone just wants to burn some money?